src/navigations/root-stack.js

```diff
@@ -152,182 +161,318 @@ const AuthenticatedNavigator = () => {
+              <Blocked {...props} />
+            </KeyboardWrapper>
+          )}
           options={{
             headerShown: false
           }}
         />
         <AuthenticatedStack.Screen
           name="GroupSetting"
-          component={GroupSetting}
+          component={(props) => (
+            <KeyboardWrapper>
+              <GroupSetting {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="AddParticipant"
-          component={AddParticipant}
+          component={(props) => (
+            <KeyboardWrapper>
+              <AddParticipant {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="GroupMedia"
-          component={GroupMedia}
+          component={(props) => (
+            <KeyboardWrapper>
+              <GroupMedia {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="GroupInfo"
-          component={GroupInfo}
+          component={(props) => (
+            <KeyboardWrapper>
+              <GroupInfo {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="DetailGroupImage"
-          component={DetailGroupImage}
+          component={(props) => (
+            <KeyboardWrapper>
+              <DetailGroupImage {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ChatDetailPage"
-          component={ChatDetailPage}
+          component={(props) => (
+            <KeyboardWrapper>
+              <ChatDetailPage {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="ReplyComment"
-          component={ReplyComment}
+          component={(props) => (
+            <KeyboardWrapper>
+              <ReplyComment {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
 
         <AuthenticatedStack.Screen
           name="ProfileReplyComment"
-          component={ProfileReplyComment}
+          component={(props) => (
+            <KeyboardWrapper>
+              <ProfileReplyComment {...props} />
+            </KeyboardWrapper>
+          )}
           options={{headerShown: false}}
         />
         <AuthenticatedStack.Screen
           name="OtherProfileReplyComment"
-          component={OtherProfileReplyComment}
+          component={(props) => (
+            <KeyboardWrapper>
+              <OtherProfileReplyComment {...props} />
+            </KeyboardWrapper>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The wrapping of each screen component with the `KeyboardWrapper` in the `AuthenticatedStack.Screen` components is done correctly. This ensures that the keyboard does not overlap with the content on each of these screens. However, it's worth noting that this approach introduces a lot of repetitive code. Consider creating a higher-order component (HOC) that automatically wraps any given component with `KeyboardWrapper`. Here's an example:

```jsx
const withKeyboardWrapper = (Component) => {
  return (props) => (
    <KeyboardWrapper>
      <Component {...props} />
    </KeyboardWrapper>
  );
};

// Usage
<AuthenticatedStack.Screen name="Settings" component={withKeyboardWrapper(Settings)} options={{headerShown: false}}/>
```

This way, you can avoid repeating the same wrapping logic for every screen.


<!-- This is an auto-generated comment by OSS CodeRabbit -->